assets/src/scripts/charcoal/admin/widget/form.js

```diff
@@ -50,6 +50,7 @@ Charcoal.Admin.Widget_Form.prototype.set_properties = function (opts) {
     this.group_conditions = opts.data.group_conditions;
     this.$form            = $(this.form_selector);
     this.allow_reload     = opts.data.allow_reload;
+    this.force_refresh     = opts.data.force_refresh;
```

Comment:
I think we should rename this property to `force_page_refresh` to distinguish from `allow_reload` which should have been named `force_widget_reload` (to be resolved in a future changeset).

```suggestion
    this.force_page_refresh = opts.data.force_page_refresh;
```

Don't forget to fix alignment of `=` assignment operators.